app/celery/tasks.py

```diff
@@ -245,7 +248,9 @@ def save_smss(self, service_id: Optional[str], signed_notifications: List[Signed
 
         reply_to_text = ""  # type: ignore
         if sender_id:
-            reply_to_text = dao_get_service_sms_senders_by_id(service_id, sender_id).sms_sender
+            reply_to_text = try_validate_and_format_phone_number(
+                dao_get_service_sms_senders_by_id(service_id, sender_id).sms_sender
+            )
```

Comment:
The old code is wrong - later on we ignore the reply_to_text if it doesn't start with a "+".